packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -218,9 +223,15 @@ export class InterfaceVpcEndpointService implements IInterfaceVpcEndpointService
    */
   public readonly port: number;
 
+  /**
+   * Whether Private DNS is supported by default.
+   */
+  public readonly privateDnsDefault?: boolean;
```

Comment:
Value can be assigned here directly, doesn't need to wait for constructor.

Just write:

```
  public readonly privateDnsDefault?: boolean = false
```

Reply:
Done